docs/notebooks/Getting_started_2.ipynb

```diff
@@ -0,0 +1,662 @@
+    "![noise.png](../assets/noise.PNG)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d4125128-de5d-4267-83ae-892e9d8ced69",
+   "metadata": {},
+   "source": [
+    "### Adversarial robustness of 1-Lipschitz neural network\n",
+    "The adversarial robustness of a predictive model is its ability to remain accurate and reliable when subjected to adversarial perturbations.  \n",
+    "\n",
+    "Practically, training an adversarial robust predictive model consists in obtaining a formal guarantee that it remains accurate, even when exposed against adversarial attacks that are defined within certain specified constraints.\n",
+    "\n",
+    "A major advantage of 1-Lipschitz neural networks is that they can offer provable guarantees on their robustness for any particular input $x$, by providing a *certificate* $\\epsilon_x$.   \n",
+    "Such a guarantee can be understood by using the following terminology:\n",
+    "\n",
+    "> \"For an input $x$, we can certify that there are no adversarial perturbations constrained to be under the certificate $\\epsilon_x$ that will change our model's prediction.\"\n",
+    "\n",
+    "In simple mathematical terms:  \n",
+    "\n",
+    "For a given $x$, $\\forall \\epsilon$ such that $||\\epsilon||<\\epsilon_x$, we obtain that:\n",
+    "$$A(x)=y,$$\n",
+    "$$A(x+\\epsilon)=y_{\\epsilon},$$\n",
+    "then:\n",
+    "$$y_{\\epsilon}=y.$$\n",
+    "\n",
+    "💡 We will not generate certificates in this notebook. However, we feel that understanding certificates offers an effective avenue for acquiring insights into the nature of adversarial robustness in the context of 1-Lipschitz neural networks. This is why we present it here.    \n",
+    "\n",
+    "💡 Depending on the type of norm you choose (e.g. L1 or L2), the guarantee you can offer will differ, as $||\\epsilon||_{L2}<\\epsilon_x$ and $||\\epsilon||_{L1}<\\epsilon_x$ are not equivalent.  \n",
+    "As such, two additional examples of guarantees with a more precise formulation would be:\n",
+    "> \"For an input $x$, we can certify that are no adversarial perturbations constrained to be within a $\\text{L2}$-norm ball of certificate $\\epsilon_{x,\\text{L2}}$ that will change our model's prediction.\"\n",
+    "\n",
+    "For a given $x$, $\\forall \\epsilon$ such that $||\\epsilon||_{L2}<\\epsilon_{x,\\text{L2}}$, we obtain that:\n",
+    "$$A(x)=y,$$\n",
+    "$$A(x+\\epsilon)=y_{\\epsilon},$$\n",
+    "then:\n",
+    "$$y_{\\epsilon}=y.$$\n",
+    "\n",
+    "> \"For an input $x$, we can certify that are no adversarial perturbations constrained to be within a $\\text{L1}$-norm ball of certificate $\\epsilon_{x,\\text{L1}}$ that will change our model's prediction.\"\n",
+    "\n",
+    "For a given $x$, $\\forall \\epsilon$ such that $||\\epsilon||_{L1}<\\epsilon_{x,\\text{L1}}$, we obtain that:\n",
+    "$$A(x)=y,$$\n",
+    "$$A(x+\\epsilon)=y_{\\epsilon},$$\n",
+    "then:\n",
+    "$$y_{\\epsilon}=y.$$\n",
```

Comment:
Done:

Added: 🚨 Note: deel-lip only deals with L2 norm

Added in the first notebook:
💡 The norm we refer to throughout our notebooks is the Euclidean norm (L2). This is because deel-lip operates with this norm. You will find more information about the role of the norm in the context of adversarially robust 1-Lipschitz deep learning models in the notebook titled 'Getting Started 2'.